tools/templates/IotDeviceBindingTemplate/dotnet_new_iot-device-binding_csharp/.template.config/template.json

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "IoT Device Binding",
+  "classifications": [ "IoT", "Console", "Library", "Test" ],
+  "identity": "DotNet.IotDeviceBinding.CSharp",
+  "name": ".NET Core IoT Device Binding Project",
+  "shortName": "iot-device-binding",
```

Comment:
Should we change the name to just device-binding? Or maybe just binding?

Reply:
Yeah, I struggled with different naming, but I kept circling back to "what does the future look like as ecosystem grows" and could there be other things labeled as "binding".  I think "iot-device-binding" kind of sticks out with the "branding" of it.  Not sure.

Reply:
If this was being added to the main dotnet sdk I would totally agree, but given that in order to use this for now you have to already be on the repo and get the template, I think that binding is short and clear enough. I agree that if this makes it to the sdk, then we should probably change the short name, but given this is very specific template for our repo, I doubt that will happen.

Reply:
Agreed.  If possible, I would still prefer do use "device-binding" as we already reference that name throughout repo.  Also, "binding" alone could be a separate meaning in a completely different topic and short name could be used in another type of template.  I'll go ahead and remove the "iot" reference throughout if that sounds good with you.